goodreads-predictor/src/goodreads_predictor/pipelines/word_embeddings/nodes.py

```diff
@@ -1,4 +1,12 @@
-"""
-This is a boilerplate pipeline 'word_embeddings'
-generated using Kedro 0.19.6
-"""
+# nodes.py
+
+import pandas as pd
+from sentence_transformers import SentenceTransformer
+
+def create_embeddings(books_loaded):
+    model = SentenceTransformer('all-MiniLM-L6-v2')
+    df = pd.read_parquet(books_loaded)
```

Comment:
**I think that the problem might come from there**

Reply:
Yep! The read_parquet is being handled natively by Kedro. You don't need to do any input/output for your functions. Just specify them in the catalog.yml file

Reply:
Updated with commits below